README.md

```diff
@@ -1,2 +1,5 @@
-# orgaquant
-orgaquant
+# OrgaQuant
+
+Implementing the OrgaQuant pipeline.
+
+Source: https://www.nature.com/articles/s41598-019-48874-y
```

Comment:
Can you add a notice that the original code was released under the MIT license, and add a copy of the license file as well? If you call it `LICENSE` and put it in the root of the repo, GitHub will pick it up automatically.

Reply:
Right, I forgot about that!